### PR TITLE
OPENEUROPA-1803: Create new Contextual navigation paragraph type.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: config/install/paragraphs.paragraphs_type.oe_contextual_navigation.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+langcode: en
+status: true
+dependencies: {  }
+id: oe_contextual_navigation
+label: 'Contextual navigation'
+icon_uuid: null
+description: ''
```
Comment (upchuk) [inline config/install/paragraphs.paragraphs_type.oe_contextual_navigation.yml]: Let's add a description like to the rest:

```
The contextual navigation allows editors to add a number of links that helps the user navigate the page. The links, their visibility limit and navigation label can be configured using the available fields.
```

File: config/install/field.field.paragraph.oe_contextual_navigation.field_oe_limit.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_limit
+    - paragraphs.paragraphs_type.oe_contextual_navigation
+id: paragraph.oe_contextual_navigation.field_oe_limit
+field_name: field_oe_limit
+entity_type: paragraph
+bundle: oe_contextual_navigation
+label: Limit
+description: ''
```
Comment (upchuk) [inline config/install/field.field.paragraph.oe_contextual_navigation.field_oe_limit.yml]: Description:

```
The number of items to show by default.
```

File: config/install/field.field.paragraph.oe_contextual_navigation.field_oe_text.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_text
+    - paragraphs.paragraphs_type.oe_contextual_navigation
+id: paragraph.oe_contextual_navigation.field_oe_text
+field_name: field_oe_text
+entity_type: paragraph
+bundle: oe_contextual_navigation
+label: 'More label'
+description: ''
```
Comment (upchuk) [inline config/install/field.field.paragraph.oe_contextual_navigation.field_oe_text.yml]:
```
The label for the link that displays all the links.
```

File: config/optional/language.content_settings.paragraph.oe_contextual_navigation.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+dependencies:
+  config:
+    - paragraphs.paragraphs_type.oe_contextual_navigation
+  module:
+    - content_translation
+third_party_settings:
+  content_translation:
+    enabled: true
+    bundle_settings:
+      untranslatable_fields_hide: '1'
```
Comment (upchuk) [inline config/optional/language.content_settings.paragraph.oe_contextual_navigation.yml]: Nice!

File: config/install/field.field.paragraph.oe_contextual_navigation.field_oe_limit.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_limit
+    - paragraphs.paragraphs_type.oe_contextual_navigation
+id: paragraph.oe_contextual_navigation.field_oe_limit
+field_name: field_oe_limit
+entity_type: paragraph
+bundle: oe_contextual_navigation
+label: Limit
+description: 'The number of items to show by default.'
```
Comment (brummbar) [inline config/install/field.field.paragraph.oe_contextual_navigation.field_oe_limit.yml]:
```suggestion
description: 'The number of items to display. When empty, defaults to 5.'
```

File: config/install/field.field.paragraph.oe_contextual_navigation.field_oe_text.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_text
+    - paragraphs.paragraphs_type.oe_contextual_navigation
+id: paragraph.oe_contextual_navigation.field_oe_text
+field_name: field_oe_text
+entity_type: paragraph
+bundle: oe_contextual_navigation
+label: 'More label'
+description: 'The label for the link that displays all the links.'
```
Comment (brummbar) [inline config/install/field.field.paragraph.oe_contextual_navigation.field_oe_text.yml]:
```suggestion
description: 'The label for the link that displays all the items. When empty, defaults to "More".'
```

**Pull request merged**